core/src/main/java/me/lokka30/treasury/plugin/core/command/subcommand/economy/EconomyInfoSub.java

```diff
@@ -56,7 +56,7 @@ public void execute(@NotNull final CommandSource sender, @NotNull final String l
                                     .getSupportedOptionalEconomyApiFeatures()
                                     .contains(OptionalEconomyApiFeature.BUKKIT_TRANSACTION_EVENTS))
                     ),
-                    placeholder("primary-currency", provider.getPrimaryCurrency().getPrimaryCurrencyName())
```

Comment:
I think we should use the currency ID here rather than the currency's display name.